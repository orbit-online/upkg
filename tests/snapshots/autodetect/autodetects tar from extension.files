[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme.bin -> ../.packages/acme@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160/bin/acme.bin
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  acme@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160
│   │       ├── [drwxr-xr-x]  bin
│   │       │   └── [-rwxr-xr-x]  acme.bin
│   │       └── [-rw-r--r--]  upkg.json
│   └── [lrwxrwxrwx]  acme -> .packages/acme@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160
└── [-rw-r--r--]  upkg.json
