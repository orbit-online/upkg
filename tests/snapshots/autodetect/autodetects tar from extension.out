upkg: Guessing pkgtype is tar, based on the URL being a file that exists on the machine and it ending in .tar or .tar.*
upkg: No checksum given for '$BATS_RUN_TMPDIR/package-fixtures/default/acme.tar.bzip2', determining now
upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/default/acme.tar.bzip2'
